### ISSUE TITLE
no stream information available

### ISSUE THREAD

**Issue by Sammeeey** [pytube/pytube#540]
Hi,
I was trying to download [this Video](https://www.youtube.com/watch?v=ZRCdORJiUgU) as described in the [quickstart](https://python-pytube.readthedocs.io/en/latest/user/quickstart.html) of the pytube-guide.

**Why are there no information about the video formats available for some videos?**

_This is what happens when I follow the instructions in the quickstart:_

```
Python 3.7.1 (default, Dec 10 2018, 22:54:23) [MSC v.1915 64 bit (AMD64)] on win32

from pytube import YouTube

yt = YouTube('https://www.youtube.com/watch?v=ZRCdORJiUgU')

yt.title
Out[4]: 'Google — Year in Search 2019'
```
Everything fine until here...

```
yt.thumbnail_url
Traceback (most recent call last):
  File "C:\Users\Samuel\Anaconda3\lib\site-packages\IPython\core\interactiveshell.py", line 3267, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-5-f4a3e63791c4>", line 1, in <module>
    yt.thumbnail_url
  File "C:\Users\Samuel\Anaconda3\lib\site-packages\pytube\__main__.py", line 261, in thumbnail_url
    return self.player_config_args['thumbnail_url']
KeyError: 'thumbnail_url'
```
KeyError: 'thumbnail_url' - currently not my main problem

But the following results are a problem...
```
yt.streams.all()
Out[6]: []

stream = yt.streams.first()

stream.download()
Traceback (most recent call last):
  File "C:\Users\Samuel\Anaconda3\lib\site-packages\IPython\core\interactiveshell.py", line 3267, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-8-0a171c2b4d83>", line 1, in <module>
    stream.download()
AttributeError: 'NoneType' object has no attribute 'download'
```

When I search for formats online with [this tool](https://www.h3xed.com/blogmedia/youtube-info.php) there are no formats available for my target video.
So yt.streams.all() only returns an empty list.
Therefore the .download command obiously does not work.

**Comment by movraxrsp** [comment]
[After applying the fix from here, I was able to download v=ZRCdORJiUgU.](https://github.com/nficano/pytube/pull/537/files/bceb929e143caadd874955fa422f8a58955bafaf#diff-fdaf06741560d8263465740e416624f0R107)

**Comment by Sammeeey** [comment]
Still I am not able to download this video with pytube.
And I still don't get stream data for it, which would be a good reason why I can not download it.

@movraxrsp did you actually do the [same as I did ](https://github.com/nficano/pytube/issues/540#issue-543503578)?
Or did I do something wrong?

**Comment by movraxrsp** [comment]
Hi @Sammeeey, yes, I did do the same as you. I did reproduce the thumbnail key error. But the download still worked.

Did you replace the files compat.py and mixins.py with the new files from the #537 pull request?

```
>>> from pytube import YouTube
>>> yt = YouTube('https://www.youtube.com/watch?v=ZRCdORJiUgU')
>>> yt.title
'Google — Year in Search 2019'
>>>
>>>
>>> stream = yt.streams.first()
>>> stream # just to check the first stream object exists
<Stream: itag="18" mime_type="video/mp4" res="360p" fps="30fps" vcodec="avc1.42001E" acodec="mp4a.40.2">
>>>
>>>
>>> yt.streams.all()
[<Stream: itag="18" mime_type="video/mp4" res="360p" fps="30fps" vcodec="avc1.42001E" acodec="mp4a.40.2">, <Stream: itag="22" mime_type="video/mp4" res="720p" fps="30fps" vcodec="avc1.64001F" acodec="mp4a.40.2">, <Stream: itag="43" mime_type="video/webm" res="360p" fps="30fps" vcodec="vp8.0" acodec="vorbis">, """omitted here for brevity"""]
>>> stream.download()
'C:\\Users\\user\\Google — Year in Search 2019.mp4'
>>>
```

**Comment by Sammeeey** [comment]
I replaced the files but it still does not work for me.

**Comment by Sammeeey** [comment]
I am getting stream information (again) for all videos now!

Some time ago I did the fix recommended in #528 .
After deleting these changes it now works for me.
